docs/components/table-view.md

```diff
@@ -0,0 +1,1016 @@
+
+<KTableView
+  :data="basicData"
+  :headers="basicHeaders()"
+>
+  <template #tooltip-email>
+    HubSpot Id: <code>8576925e-d7e0-4ecd-8f14-15db1765e69a</code>
+  </template>
+</KTableView>
+
+```html
+<KTableView
+  :data="tableData"
+  :headers="headers"
+>
+  <template #tooltip-email>
+    HubSpot Id: <code>8576925e-d7e0-4ecd-8f14-15db1765e69a</code>
+  </template>
+</KTableView>
+```
+
+### toolbar
+
+Toolbar is rendered directly above the table. Useful for slotting table controls like search or filter fields.
+
+:::tip NOTE
+Toolbar slot container is a `display: flex;` element that takes care of spacing between slotted items so you can slot in your items without having to worry about adding margin between them.
+:::
+
+<KTableView
+  :data="basicData"
+  :headers="basicHeaders()"
+>
+  <template #toolbar>
+    <KInput placeholder="Search">
+      <template #before>
+        <SearchIcon />
+      </template>
+    </KInput>
+    <KButton size="large">
+      <AddIcon /> Add user
+    </KButton>
+  </template>
+</KTableView>
+
+```html
+<KTableView
+  :data="tableData"
+  :headers="headers"
+>
+  <template #toolbar>
+    <KInput placeholder="Search">
+      <template #before>
+        <SearchIcon />
+      </template>
+    </KInput>
+    <KButton size="large">
+      <AddIcon /> Add user
+    </KButton>
+  </template>
+</KTableView>
+```
+
+### empty-state
+
+Slot content to be displayed when empty.
+
+### empty-state-action-icon
+
+Slot for icon to be displayed in front of action button text in empty state. See [empty state](#empty) section for example of usage of this slot.
+
+### error-state
+
+Slot content to be displayed when in error state.
+
+### actions-items
+
+:::warning NOTE
+This slot is only available in KTableView and not available in [KTable](/components/table).
+:::
+
+Slot for passing actions dropdown items. See [KDropdownItem component docs](/components/dropdown#kdropdownitem) for details.
+
+Slot props:
+* `row` - table row object
+* `rowKey` - table row index
+
+:::tip NOTE
+This slot is only available when `actions` header hey is present in [`headers` prop](#reserved-header-keys).
+:::
+
+<KTableView
+  :data="basicData"
+  :headers="basicHeaders(true)"
+>
+  <template #actions-items>
+    <KDropdownItem>
+      Edit
+    </KDropdownItem>
+    <KDropdownItem
+      danger
+      has-divider
+    >
+      Delete
+    </KDropdownItem>
+  </template>
+</KTableView>
+
+```html
+<KTableView
+  :data="tableData"
+  :headers="headers"
+>
+  <template #actions-items>
+    <KDropdownItem>
+      Edit
+    </KDropdownItem>
+    <KDropdownItem
+      danger
+      has-divider
+    >
+      Delete
+    </KDropdownItem>
+  </template>
+</KTableView>
+```
+
+### after
+
+:::warning NOTE
+This slot is only available in KTableView and not available in [KTable](/components/table).
+:::
+
+Slot for content that should be rendered below the table. For example, could be used for displaying pagination component.
```

Comment:
Is this slot still needed now that we're baking in pagination component?

Reply:
Yes - part of component brief.

Reply:
what would a user slot in below the pagination?

Reply:
Maybe pagination that's not KPagination? Can't think of our internal use case though. Remove the slot?

Reply:
No we have a `hidePagination` prop

Reply:
So if a user wanted to utilize their own KPagination (or something) instead of the built-in, do they need a slot, or can they just place it under the table? 

If they don't need the slot, let's just remove it.